app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -455,6 +456,39 @@ class PatientRepositoryAndroidTest {
     assertThat(searchResult).isEmpty()
   }
 
+  /*
+  * Added to test the case where SQLite's max query param length (999) can be
+  * exceeded during fuzzy name search.
+  **/
+  @Test
+  fun when_searching_with_fuzzy_search_the_results_must_be_limited() {
+    val template = testData.patientProfile(syncStatus = SyncStatus.DONE)
+
+    val patientsToSave = (1 .. 1000).map {
```

Comment:
`1000` could potentially be replaced with `sqliteParamLimit + 1`, where `sqliteParamLimit = 999` just for readability instead of depending on the function documentation.

Reply:
Good point. I'll fix this in my next PR.

Reply:
🙂👍